depends/packages/packages.mk

```diff
@@ -1,4 +1,8 @@
-packages:=boost libevent
+packages:=
```

Comment:
Concept ACK
Nit: I guess this could be removed here and `packages` be defined in `depends/Makefile`. Not sure if this is actually better though.

Reply:
I think we'll just leave this here for now.